Project.toml

```diff
@@ -0,0 +1,20 @@
+
+name = "Xpress"
+uuid = "9e70acf3-d6c9-5be6-b5bd-4e2c73e3e054"
+authors = ["Dheepak Krishnamurthy <me@kdheepak.com>"]
+version = "0.9.1"
+
+[deps]
+CEnum = "fa961155-64e5-5f13-b03f-caf6b980ea82"
+Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+
+[compat]
+CEnum = "^0.2"
+DocStringExtensions = "^0.8"
```

Comment:
Do we really need this?

Reply:
I've previously used it for API documentation in other packages and found it useful. So you can add a minimal docstring in the [source code](https://github.com/GMLC-TDC/HELICS.jl/blob/f421dc45df489ed1fb87a41d67d957f046a7a1f7/src/api.jl#L839-L844) and it will generate the [function signature in the documentation](https://gmlc-tdc.github.io/HELICS.jl/latest/api/#HELICS.helicsGetVersion-Tuple{}). I can remove it if you'd like to not have this dependency.

Reply:
I'd say that we should not use packages that are not used by JuMP, MOI, CPLEX and Gurobi.
Better for robusteness.
FICO's manual is online so, at leat most o the C API is covered

Reply:
Resolved in https://github.com/JuliaOpt/Xpress.jl/pull/43/commits/42319d303bae7cfbd62e76e0c2a19a0600f895cb.